pkg/types/interfacetests/chain_reader_interface_tests.go

```diff
@@ -88,20 +90,7 @@ func runChainReaderGetLatestValueInterfaceTests[T TestingT[T]](t T, tester Chain
 			},
 		},
 		{
-			name: "Get latest value allows a contract name to resolve different contracts internally",
```

Comment:
Why eliminate this test?

Reply:
It doesn't test what it says and the thing that it actually tests already has a test

Reply:
my bad, I returned the test, it makes sense in the evm impl.